static/src/stylesheets/amp/_hosted.scss

```diff
@@ -0,0 +1,853 @@
+    width: 80px;
+    z-index: 1015;
+}
+
+.hostedbadge:focus {
+    text-decoration: none;
+}
+
+.hostedbadge--pushdown {
+    top: 0;
+}
+
+.hostedbadge__info {
+    font-size: 14px;
+    font-weight: 600;
+    letter-spacing: .5px;
+    line-height: 16px;
+    height: auto;
+    padding-top: 5px;
+    padding-bottom: 8px;
+    color: #333333;
+    margin: 0;
+    text-align: center;
+}
+
+.hostedbadge .hostedbadge__logo {
+    position: absolute;
+    display: block;
+    width: 100%;
+    top: 100%;
+    background-color: #ffffff;
+}
+
+.hosted-header__spacer {
+    flex: 1;
+}
+
+.hosted__video video {
+    width: 100%;
+}
+
+.hosted-page {
+    .content__article-body {
+        margin: 0;
+    }
+
+    .content__article-body .media-primary {
+        margin-left: -10px;
+    }
+
+    .u-responsive-ratio img {
+        width: 100%;
+        height: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+    }
+
+    .content .gs-container {
+        padding: 0 10px;
+    }
+}
+
+.hosted-article-page {
+    .content-article-body {
+        margin: 0;
+    }
+
+    .media-primary.media-primary--showcase {
+        position: relative;
+        height: 378px;
+        margin-bottom: 10px;
+        background-size: cover;
+        background-position: center center;
+        &::after {
+            content: '';
+            display: block;
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            z-index: 2;
+            background: linear-gradient(to top, rgba(51, 51, 51, .4) 0%, rgba(51, 51, 51, .4) 2%, transparent 100%);
+        }
+    }
+
+    &.hosted-page--bright .content__article-body p.intro + p::first-letter {
+        color: #ffffff;
+    }
+
+    .gs-container {
+        height: 100%;
+    }
+
+    .title {
+        position: absolute;
+        margin: auto;
+        z-index: 3;
+        bottom: 20px;
+        color: #ffffff;
+        font-size: 32px;
+        line-height: 36px;
+        font-weight: bold;
+    }
+
+    .caption {
+        position: absolute;
+        left: 10px;
+        top: 390px;
+        right: unset;
+        padding: 0;
+    }
+
+    figure {
+        margin-left: 0;
+        margin-right: 0;
+    }
+
+    figcaption {
+        font-size: 0.75rem;
+        line-height: 1rem;
+        color: #767676;
+    }
+
+    .content__meta-container {
+        margin: 48px 0 12px 0;
+    }
+
+    .meta__extras {
+        border-top: none;
+    }
+
+    .social__item--pinterest {
+        display: inline-block;
+    }
+
+    .content__article-body p {
+        font-size: 18px;
+        line-height: 24px;
+    }
+
+    .content__article-body p.intro + p::first-letter {
+        display: inline-block;
+        float: left;
+        color: #000000;
+        padding: 12px 16px 4px;
+        font-size: 480%;
+        line-height: 4.25rem;
+        width: 4.8125rem;
+        height: 4.8125rem;
+        margin-right: .5rem;
```

Comment:
Probably best to use `px` for code consistency. It will be converted to `rem` values anyway